src/main/resources/js/condec.dialog.js

```diff
@@ -36,15 +36,21 @@
 		console.log("view.context.menu.js setUpDialogForCreateAction");
 		console.log(id);
 		setHeaderText(createKnowledgeElementText);
-		setUpCreateOrEditDialog("", "", "Alternative");
+		setUpCreateOrEditDialog("", "", "Alternative",true);
 
 		var submitButton = document.getElementById("dialog-submit-button");
 		submitButton.textContent = createKnowledgeElementText;
 		submitButton.onclick = function() {
 			var summary = document.getElementById("form-input-summary").value;
 			var description = document.getElementById("form-input-description").value;
 			var type = $("select[name='form-select-type']").val();
-			conDecAPI.createDecisionKnowledgeElementAsChild(summary, description, type, id);
+			var documentationLocation = $("select[name='form-select-location']").val();
+			if(documentationLocation === "i"){
```

Comment:
I think, we should use more speaking names for the documentation location on frontend to improve the readabilty of code. Only the ids in the link database should use the one-character-prefixes.

Reply:
You're right!
This is because i thought of the JSON Parser for the constructor of DocumentLocation. Therefore this needed to be "i" and "s". But since this is implicit clear with the rest method, it can be changed.